test/ConstantProduct.t.sol

```diff
@@ -3,5 +3,6 @@ pragma solidity ^0.8.13;
 
 import {VerifyTest} from "./ConstantProduct/VerifyTest.sol";
 import {GetTradeableOrderTest} from "./ConstantProduct/GetTradeableOrderTest.sol";
+import {Ierc165Test} from "./ConstantProduct/Ierc165Test.sol";
 
-contract ConstantProductTest is VerifyTest, GetTradeableOrderTest {}
+contract ConstantProductTest is VerifyTest, GetTradeableOrderTest, Ierc165Test {}
```

Comment:
nit: In general, I'd be a fan of retaining capitalised `I` for interfaces, and all instances of `ERC` being capitalised in contract names. [See solidity style guide](https://docs.soliditylang.org/en/latest/style-guide.html#naming-styles).
```suggestion
contract ConstantProductTest is VerifyTest, GetTradeableOrderTest, IERC165Test {}
```